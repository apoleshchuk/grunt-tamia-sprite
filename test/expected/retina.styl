retina_en = 0px 0px 14px 11px
retina_ru = -17px 0px 14px 11px
retina_width = 31px
retina_height = 11px