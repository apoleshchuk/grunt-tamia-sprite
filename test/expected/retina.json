{"properties":{"width":31,"height":11},"coordinates":{"en":{"x":0,"y":0,"width":14,"height":11},"ru":{"x":-17,"y":0,"width":14,"height":11}}}